lib/dor/workflow/client/status.rb

```diff
@@ -99,12 +102,14 @@ def simplified_status_code(display)
 
         def current_milestones
           current = []
-          # only get steps that are part of accessioning and part of the current version. That can mean they were archived with the current version
-          # number, or they might be active (no version number).
-          milestones.each do |m|
-            if STEPS.key?(m[:milestone]) && (m[:version].nil? || m[:version] == version)
-              current << m unless m[:milestone] == 'registered' && version.to_i > 1
-            end
+          # only get steps that are part of accessioning and part of the current version.
+          # That can mean they were archived with the current version number, or they might be active (no version number).
+          milestones.each do |mstone|
+            next unless STEPS.key?(mstone[:milestone]) && (mstone[:version].nil? || mstone[:version] == version)
```

Comment:
In the above code, we only get inside the conditional if both are true. Since the `if` is changing to an `unless`, does that suggest the `&&` here should be an `||`?

Side thought... how well tested is this method anyway?